### ISSUE TITLE
-Wenum-conversion in drivers/gpu/drm/amd/amdkfd/kfd_packet_manager_v9.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1596]
```
drivers/gpu/drm/amd/amdgpu/../amdkfd/kfd_packet_manager_v9.c:267:3: error: implicit conversion from enumeration type 'enum mes_map_queues_extended_engine_sel_enum' to different enumeration type 'enum mes_unmap_queues_extended_engine_sel_enum' [-Werror,-Wenum-conversion]
                extended_engine_sel__mes_map_queues__sdma0_to_7_sel :
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Patch submitted: https://lore.kernel.org/r/20220217162142.1828990-1-nathan@kernel.org/

**Comment by nathanchance** [comment]
Patch accepted: https://gitlab.freedesktop.org/agd5f/linux/-/commit/b63c54d978236dd6014cf2ffba96d626e97c915c

**Comment by nathanchance** [comment]
Fixed in `next-20220222`: https://git.kernel.org/next/linux-next/c/b63c54d978236dd6014cf2ffba96d626e97c915c
